docs/content/dagster-cloud/getting-started/getting-started-with-hybrid-deployment.mdx

```diff
@@ -0,0 +1,172 @@
+
+We'll walk you through setting up your Dagster Cloud account, setting up an agent, loading your code, launching your first run, and configuring continuous integration.
+
+---
+
+## Prerequisites
+
+To complete the steps in this guide, you'll need:
+
+- **Access to some Dagster code.** You can either use your own code or our [sample code (`dagster-cloud-examples`)](https://github.com/dagster-io/dagster-cloud/tree/main/dagster-cloud-examples). This guide uses the `dagster-cloud-examples` Python package.
+
+- **The ability to create and configure an agent in your infrastructure.** Using a Hybrid deployment requires setting up an agent for use with Dagster Cloud. The agent runs in your infrastructure, executes your code, and streams metadata about code execution over HTTPS back to Dagster Cloud.
+
+  Dagster Cloud supports a handful of agents, each with its own prerequisites for setup. Refer to the [Agent documentation](/dagster-cloud/deployment/agents) for more info.
+
+---
+
+## Step 1: Create a Dagster Cloud account and organization
+
+If you don't already have a Dagster Cloud account, [sign up for one](https://dagster.cloud/) before continuing.
+
+You'll also be asked to create a Dagster Cloud organization. When your organization is created, a single deployment, named `prod`, will also be created. Deployments are standalone environments, allowing you to operate independent instances of Dagster with separately managed permissions.
+
+---
+
+## Step 2: Select Hybrid as your deployment type
+
+1. When prompted, click **Select** under **Hybrid deployment**.
+2. The modal that displays will contain a pre-generated [agent token](/dagster-cloud/account/managing-user-agent-tokens) and info about the agents currently supported by Dagster Cloud.
+3. Follow the steps for setting up and deploying your agent:
+
+   - [Amazon Elastic Container Service (ECS)](/dagster-cloud/deployment/agents/amazon-ecs/creating-ecs-agent-new-vpc)
+   - [Docker](/dagster-cloud/deployment/agents/docker)
+   - [Kubernetes](/dagster-cloud/deployment/agents/kubernetes/configuring-running-kubernetes-agent)
+   - [Local](/dagster-cloud/deployment/agents/local)
+
+Continue to the next step when finished.
+
+---
+
+## Step 3: Deploy your code
+
+Now that you've set up your deployment, you'll need to tell it the location of your Dagster code.
+
+A code location specifies a single Python package or file that defines your Dagster code. Each Dagster Cloud deployment can include code from one or more code locations. Refer to the [Adding a code location guide](/dagster-cloud/developing-testing/code-locations) for more info.
+
+To complete this step, you can use your own Dagster code or the [`dagster-cloud-examples` code](https://github.com/dagster-io/dagster-cloud/tree/main/dagster-cloud-examples). Our examples use the `dagster-cloud-examples` package.
```

Comment:
To complete this step, you can use your own Dagster code or a project scaffold. Our examples use the scaffold. You can scaffold a new project by running:

```
dagster project scaffold \
  --name my-dagster-project \
  --example assets_modern_data_stack
```